src/Client.cpp

```diff
@@ -673,6 +682,11 @@ ::doHandler()
 
 		cleanupAfterLoop();
 	}
+#ifdef USING_VMA_EXTRA_API
```

Comment:
i agree but this can cause a potential memory leak if the init function fails and cleanupAfterLoop is not called